app/views/ProjectEdit/Users/UserList/AddUserModal/index.tsx

```diff
@@ -110,41 +152,89 @@ function AddUserModal(props: Props) {
     });
 
     const {
-        pending: pendingAddAction,
-        trigger: triggerAddProjectMember,
-    } = useLazyRequest<unknown, ValueToSend>({
-        url: isDefined(userValue)
-            ? `server://projects/${projectId}/project-memberships/${userValue.id}/`
-            : `server://projects/${projectId}/project-memberships/`,
-        method: isDefined(userValue)
-            ? 'PATCH'
-            : 'POST',
-        body: (ctx) => ctx,
-        onSuccess: () => {
-            onTableReload();
-            onModalClose();
+        loading: badgeOptionsPending,
+        data: badgeOptionsResponse,
+    } = useQuery<UserBadgeOptionsQuery, UserBadgeOptionsQueryVariables>(
+        USER_BADGE_OPTIONS,
+    );
+    const [
+        bulkEditProjectMembership,
+        { loading: bulkEditProjectMembershipPending },
+    ] = useMutation<
+        ProjectMembershipBulkUpdateMutation,
+        ProjectMembershipBulkUpdateMutationVariables
+    >(
+        PROJECT_MEMBERSHIP_BULK,
+        {
+            onCompleted: (response) => {
+                if (!response?.project?.projectUserMembershipBulk) {
+                    return;
+                }
+                const {
+                    errors,
+                    result,
+                } = response.project.projectUserMembershipBulk;
+
+                const [err] = errors ?? [];
+                const [user] = result ?? [];
+                if (errors && errors[0]) {
```

Comment:
Do we need to check for the errors to have an item? can we just not check for length?